modules/tinymce/src/plugins/textpattern/test/ts/browser/FindInlinePatternTest.ts

```diff
@@ -24,304 +36,268 @@ UnitTest.asynctest('browser.tinymce.plugins.textpattern.FindInlinePatternTest',
         { start: 'asap', replacement: 'as soon as possible' }
       ]
   };
-
   const inlinePatterns = Settings.getPatternSet(mockEditor as any).inlinePatterns;
 
-  const cGetInlinePattern = (patterns: InlinePattern[], space: boolean = false) => {
-    const asStr = (p: InlinePattern) => {
-      if (p.type === 'inline-format') {
-        return p.start + 'TEXT' + p.end + ' = ' + JSON.stringify(p.format);
-      } else {
-        const m = (p.start === '' || p.end === '') ? p.start + p.end : p.start + 'TEXT' + p.end;
-        return m + ' = ' + p.cmd + '(' + JSON.stringify(p.value) + ')';
-      }
-    };
-
-    return Chain.label('Get inline ' + Arr.map(patterns, asStr).join(', '),
-      Chain.mapper<Editor, InlinePatternMatch[]>((editor) => {
-        return findPatterns(editor, patterns, space);
-      })
-    );
+  const getInlinePattern = (editor: Editor, patterns: InlinePattern[], space: boolean = false) =>
+    findPatterns(editor, patterns, space);
+
+  const assertPatterns = (actualMatches: InlinePatternMatch[], expectedMatches: ExpectedPatternMatch[]) => {
+    assert.lengthOf(actualMatches, expectedMatches.length, 'Pattern count does not match');
+    for (let i = 0; i < expectedMatches.length; i++) {
+      const expected = expectedMatches[i];
+      const actual = actualMatches[i];
+      const pattern = actual.pattern;
+      Obj.each(expected.pattern, (value, key) => {
+        if (Obj.has<any, string>(pattern, key)) {
+          assert.deepEqual(pattern[key], value, 'Pattern ' + (i + 1) + ' property `' + key + '` is not equal');
```

Comment:
```suggestion
          assert.deepEqual(pattern[key], value, `Pattern ${i + 1} property \`${key}\` is not equal`);
```

Reply:
There's another one just below - rabbit hole?

Reply:
Yeah, it's really out of scope for this sort of thing. I did a couple if I touched parts of it, but there's many out there.